ddd-koans/Entity/Identifier/task.html

```diff
@@ -0,0 +1,59 @@
+<html>
+<h2>Identifier</h2>
+<p>
+    In the previous parts of the course, the identification topic was briefly mentioned. The value object that is
+    identified by its value as described, however, the opposite situation was not presented.
```

Comment:
`identified` -> `identified and differentiated `

`however, the opposite situation was not presented.` -> `but what are other possibilities to differentiate between concepts?`